.github/compatibility-test-matrices/release-v7.4.x/transfer-chain-a.json

```diff
@@ -4,14 +4,12 @@
   ],
   "chain-b": [
     "v8.2.0",
-    "v8.1.0",
-    "v8.0.1",
-    "v8.0.0",
     "v7.4.0",
-    "v7.3.2",
-    "v7.3.1",
-    "v7.2.3",
-    "v7.2.2",
+    "v6.3.0",
+    "v5.4.0",
+    "v4.6.0",
+    "v3.4.0",
+    "v2.5.0",
```

Comment:
I have added here the latest tags for the previous release that are already EoL (except v1, for which there is no docker image). I think this is good to do, even if those release lines are not supported anymore, just to make sure that we keep backwards compatibility, but I am open to hear everyone's thoughts.

Reply:
I think it's strictly necessary but it's no harm, especially since some of these versions are still in use.